api/app/controllers/api/v1/passwords_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V1
+    # Passwords controller
+    class PasswordsController < ApplicationController
+      include Password
+
+      def new
+        user = User.find_by(email: params[:email])
+        return user_not_found_error unless user
+        user.process_reset_password_request
+      end
+
+      def update
+        user = User.by_reset_token(params[:reset_token]).first
```

Comment:
This is a weird one... Moving the `.first` to the `.by_reset_token` method returns what I want in the console, but returns many records in the controller.
`user = User.by_reset_token(params[:reset_token])
  puts user.count => 42`

When it's in the controller, I get the correct user.